app/controllers/logs_controller.rb

```diff
@@ -390,6 +383,20 @@ def export
 
   private
 
+  def parse_start_stop_date(params)
+    start_date_string = params['start_date']
+    stop_date_string = params['stop_date']
+
+    start_date = Date.parse(start_date_string)
+    stop_date = Date.parse(stop_date_string)
+
+    start_date_hash = Hash[month: start_date.month.to_s, day: start_date.day.to_s, year: start_date.year.to_s]
+    params['start_date'] = start_date_hash
+
+    stop_date_hash = Hash[month: stop_date.month.to_s, day: stop_date.day.to_s, year: stop_date.year.to_s]
+    params['stop_date'] = stop_date_hash
```

Comment:
Couple of notes. 

You do `params['start_date']` here and then above `params[:start_date]`. Consistency is important. 

Also `start_date_hash` seems unnecessary to have as a variable? Can you just do `params[:start_date] = Hash[month: st...` ? Same for `stop_date_hash`?